.github/workflows/go.yml

```diff
@@ -20,13 +20,9 @@ jobs:
         env:
           GO111MODULE: "on"
         run: go build ./...
-      - name: Lint
+      - name: Test
         env:
           GO111MODULE: "on"
         run: |
           export PATH=${PATH}:$(go env GOPATH)/bin
-          golangci-lint run --disable-all --deadline=10m --enable=goimports --enable=gosimple --enable=unconvert --enable=ineffassign --enable=govet --enable=structcheck
-      - name: Test
-        env:
-          GO111MODULE: "on"
-        run: go test -v -short ./...
+          sh ./run_tests.sh
```

Comment:
Calling it this way might try to run it with the wrong shell, depending on what `sh` is set as.  Given the shebang line in run_tests.sh that specifies bash, can we run this as just `./run_tests.sh`.

Reply:
good catch, rectifying.